### PR TITLE
added batman method

### PR DESCRIPTION
**Pull request by radiantnode**
My contribution to this amazing resource!

### CODE REVIEW

File: anything.js [hunk 1]
```diff
@@ -386,6 +386,11 @@
         return x;
     }
 
+    function batman ()
+    {
+      return Array(10).join('n'-1) + ' Batman!';
```
Comment (thehowl) [inline anything.js:391]: Wouldn't 16 be better?

**Comment by Rabrennie** [comment]
I was looking at it like how would that even work. Then I was like oohhhhhhh.

**Comment by thehowl** [comment]
https://youtu.be/FqhZZNUyVFM?t=3m23s

**Comment by Rabrennie** [comment]
I've seen that before totally missed the batman thing. 10/10

**Comment by radiantnode** [comment]
@thehowl Haha I was inspired by that video.

**Pull request closed without merging**